CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 2.21.0 / 2025-05-05
```

Comment:
we should update this to 2025-05-06 since it didnt go out yesterday, but other than that lgtm